docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/ui/index.md

```diff
@@ -46,4 +46,14 @@ In the image below, you can see an example of a data product. It not only provid
 
 ![Event specification details](images/event-specification-details.png)
 
-If you need to edit a data product at any time, simply select it from the data products listing accessible from the main menu.
\ No newline at end of file
+If you need to edit a data product at any time, simply select it from the data products listing accessible from the main menu.
+
+#### Updating existing Data Products following the release of Source Applications
+
+Data Products created prior to the release of [Source Applications](../../organize-data-sources-with-source-applications/index.md) will need to be updated to assign Source Applications in which the events will be tracked on. The selected Source Applications will add the relevant application IDs to each of the event specifications in the data product. 
```

Comment:
I would add a date here, for instance Data Products created prior to the release of Source Applications (September 2024)... or something like that